x-pack/plugins/security_solution/public/overview/components/user_overview/index.tsx

```diff
@@ -99,38 +99,35 @@ export const UserOverview = React.memo<UserSummaryProps>(
     );
 
     const [userRiskScore, userRiskLevel] = useMemo(() => {
-      if (isModuleEnabled) {
-        const userRiskData = userRisk && userRisk.length > 0 ? userRisk[0] : undefined;
-        return [
-          {
-            title: i18n.USER_RISK_SCORE,
-            description: (
-              <>
-                {userRiskData
-                  ? Math.round(userRiskData.user.risk.calculated_score_norm)
-                  : getEmptyTagValue()}
-              </>
-            ),
-          },
-          {
-            title: i18n.USER_RISK_CLASSIFICATION,
-            description: (
-              <>
-                {userRiskData ? (
-                  <RiskScore
-                    severity={userRiskData.user.risk.calculated_level as RiskSeverity}
-                    hideBackgroundColor
-                  />
-                ) : (
-                  getEmptyTagValue()
-                )}
-              </>
-            ),
-          },
-        ];
-      }
-      return [undefined, undefined];
-    }, [userRisk, isModuleEnabled]);
+      const userRiskData = userRisk && userRisk.length > 0 ? userRisk[0] : undefined;
+      return [
+        {
+          title: i18n.USER_RISK_SCORE,
+          description: (
+            <>
+              {userRiskData
+                ? Math.round(userRiskData.user.risk.calculated_score_norm)
+                : getEmptyTagValue()}
+            </>
+          ),
+        },
+        {
+          title: i18n.USER_RISK_CLASSIFICATION,
+          description: (
+            <>
+              {userRiskData ? (
+                <RiskScore
+                  severity={userRiskData.user.risk.calculated_level as RiskSeverity}
```

Comment:
userRiskData.user.risk.calculated_level is `RiskSeverity` already

```suggestion
                  severity={userRiskData.user.risk.calculated_level}
```